### ISSUE TITLE
WebUI has no content due to incorrect metric

### ISSUE THREAD

**Issue by colorjam** [microsoft/nni#2278]
![webui_issue](https://user-images.githubusercontent.com/23700012/78734465-f4e76d80-797a-11ea-851e-5b49bb5a832e.jpg)

**Short summary about the issue/question**:
metric contains  `str` so the WebUI cannot display content. I tried to export trials by CSV type, but got the following errors:
```
Traceback (most recent call last):
  File "/home/chenqian/miniconda3/envs/py37/bin/nnictl", line 11, in <module>
    load_entry_point('nni', 'console_scripts', 'nnictl')()
  File "/home/chenqian/workspace/nni/build/nni_cmd/nnictl.py", line 223, in parse_args
    args.func(args)
  File "/home/chenqian/workspace/nni/build/nni_cmd/nnictl_utils.py", line 703, in export_trials_data
    formated_record = {**record['parameter'], **record['value'], **{'id': record['id']}}
TypeError: 'str' object is not a mapping
```
The JOSN type output file shows that there are `-Infinity` in the metric
![image](https://user-images.githubusercontent.com/23700012/78735233-d8e4cb80-797c-11ea-9dc3-cfd2f167e677.png)

**Brief what process you are following**: 
I use `-loss` as the metric, some trials run well while the final results are`-Infinity`.

**How to reproduce it**: 

**nni Environment**:
- nni version:
- nni mode(local|pai|remote):
- OS:
- python version:
- is conda or virtualenv used?: 
- is running in docker?:

**need to update document(yes/no)**:

**Anything else we need to know**:

**Comment by liuzhe-lz** [comment]
NNI v1.5 adds support for infinity and string metrics. It should be released on today or tomorrow.

NNI cannot export non-dict metrics to CSV file at this moment. You may export to JSON file instead.

**Comment by scarlett2018** [comment]
Closing it as the issue is fixed in 1.5. @colorjam - please feel free to reopen if there are still issues after you try 1.5.
